pkg/kubelet/config/file.go

```diff
@@ -138,23 +138,23 @@ func (s *sourceFile) extractFromDir(name string) ([]*v1.Pod, error) {
 
 		switch {
 		case statInfo.Mode().IsDir():
-			glog.V(1).Infof("Not recursing into config path %q", path)
+			glog.V(1).Infof("Not recursing into manifest path %q", path)
 		case statInfo.Mode().IsRegular():
 			pod, err := s.extractFromFile(path)
 			if err != nil {
-				glog.V(1).Infof("Can't process config file %q: %v", path, err)
+				glog.V(1).Infof("Can't process manifest file %q: %v", path, err)
 			} else {
 				pods = append(pods, pod)
 			}
 		default:
-			glog.V(1).Infof("Config path %q is not a directory or file: %v", path, statInfo.Mode())
+			glog.V(1).Infof("Manifest path %q is not a directory or file: %v", path, statInfo.Mode())
```

Comment:
I think the comments of this func explain the reason. These cases aren't considered as errors, so the log also doesn't use Errorf level.

>// Get as many pod configs as we can from a directory. Return an error if and only if something
  // prevented us from reading anything at all. Do not return an error if only some files
  // were problematic.

/cc @yujuhong @mtaufen